pylint_plugins/__init__.py

```diff
@@ -1,6 +1,7 @@
 from .pytest_raises_checker import PytestRaisesChecker
 from .print_function import PrintFunction
 from .unittest_assert_raises import UnittestAssertRaises
+from .string_checker import StringChecker  # pylint: disable=unused-import
```

Comment:
```suggestion
from .string_checker import StringChecker
```

@tsugumi-sys You can make this change in your PR.